### PR TITLE
Fix title and use generic includes

### PR DESCRIPTION
**Pull request by iranzo**
Signed-off-by: Pablo Iranzo Gómez <Pablo.Iranzo@gmail.com>

**What this PR does / why we need it**:

**Does this PR fix any issue?** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
> Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: pages/cloud.md [hunk 1]
```diff
@@ -42,6 +42,110 @@ Check Kubernetes.io guide for each cloud provider to match your use case:
 
 After following the instructions provided by <Kubernetes.io>, `kubectl` can be used to manage the cluster.
 
-## Step2: Go to the labs!
+### Deploy KubeVirt Operator
 
-At this point you can follow with the [Labs in this website]({% link pages/labs.md %}) to deploy and experiment with KubeVirt.
+Having the cluster is up and running, let's set the *version* environment
```
Comment (alosadagrande) [inline pages/cloud.md]: Having the cluster up and running,....

Comment (iranzo) [reply pages/cloud.md]: already changed

**Comment by alosadagrande** [comment]
@iranzo I think a rebase it is needed

**Comment by alosadagrande** [comment]
There are some rendering errors. Table is not formatted correctly and some `kubectl` word.

**Comment by iranzo** [comment]
@alosadagrande table fixed, thanks !

File: pages/cloud.md [hunk 2]
```diff
@@ -20,26 +20,130 @@ KubeVirt can be tested on external Cloud Providers like AWS, Azure, GCP, AliClou
...
 
 > info ""
 > Create a disk of 30Gb at least.
 
 After following the instructions provided by <Kubernetes.io>, `kubectl` can be used to manage the cluster.
 
-## Step2: Go to the labs!
+### Deploy KubeVirt Operator
+
+Having the cluster is up and running, let's set the _version_ environment
```
Comment (alosadagrande) [inline pages/cloud.md]: Having the cluster up and running,

File: pages/cloud.md [hunk 3]
```diff
@@ -20,26 +20,130 @@ KubeVirt can be tested on external Cloud Providers like AWS, Azure, GCP, AliClou
...
 | AliCloud | <https://kubernetes.io/docs/setup/production-environment/turnkey/alibaba-cloud/> |
 | AWS      | <https://kubernetes.io/docs/setup/production-environment/turnkey/aws/>           |
 | Azure    | <https://kubernetes.io/docs/setup/production-environment/turnkey/azure/>         |
 | GCP      | <https://kubernetes.io/docs/setup/production-environment/turnkey/gce/>           |
-| Others   | <https://kubernetes.io/docs/setup/production-environment/turnkey/>               |
 
 > info ""
 > Create a disk of 30Gb at least.
 
 After following the instructions provided by <Kubernetes.io>, `kubectl` can be used to manage the cluster.
```
Comment (alosadagrande) [inline pages/cloud.md]: kubectl is not shown as expected. Probably, <kubernetes.io> is not behaving as expected either.

File: pages/cloud.md [hunk 4]
```diff
@@ -20,26 +20,130 @@ KubeVirt can be tested on external Cloud Providers like AWS, Azure, GCP, AliClou
...
+$ kubectl krew install virt
+```
+
+Then `virtctl` can be used as a kubectl plugin. For a list of available commands run:
+
+```bash
+$ kubectl virt help
+```
+
+Every occurrence throughout this guide of
```
Comment (alosadagrande) [inline pages/cloud.md]: I understand what you want to say but it is not clear at first glance.
Probably:

Once krew plugin is installed, every occurrence throughout this guide where virtcl appears can be replaced by kubectl virt...

File: pages/cloud.md (hunk 2 above)
Comment (iranzo) [reply pages/cloud.md]: done in next review

File: pages/cloud.md (hunk 3 above)
Comment (iranzo) [reply pages/cloud.md]: updating

File: pages/cloud.md (hunk 4 above)
Comment (iranzo) [reply pages/cloud.md]: done

**Pull request merged**